src/firmware/tsdz2/lights.c

```diff
@@ -18,7 +18,7 @@ void lights_init()
 	SET_PIN_OUTPUT(PIN_LIGHTS);
 
 	lights_enabled = false;
-	lights_set(false);
+	lights_set(g_config.lights_always_on);
```

Comment:
not sure what happened here, should be false, same as bbsx

Reply:
Done